src/Action.js

```diff
@@ -66,11 +66,21 @@ function Action(action) {
 				});
 			}
 		});
-
-		this.fields = action.fields;
```

Comment:
Oh huh, this is redundant because of line 53 ja?

Reply:
It's incorrect rather than redundant. It replaces the Array of SirenField objects with the vanilla objects. Therefore, you can't do stuff like `sirenAction.fields[0].hasClass('abc')` as shown in the added test.

Reply:
Huh - I guess whenever I've used it I've not directly accessed the fields like that. Nice catch! :)